src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/debugger/Debugger.java

```diff
@@ -109,6 +109,11 @@ which this debugger is running (to be able to properly configure
       null. */
   public CDebugger getCDebugger() throws DebuggerException;
 
+  /**
+   * Find address and executable which contains symbol.
+  */
```

Comment:
Just a whitespace nit - the * doesn't align. 8-)

Reply:
Thanks! I aligned whitespace in new commit.